strawberryfields/gbs/plot.py

```diff
@@ -260,3 +261,72 @@ def plot_subgraph(subgraph: nx.Graph, size: int = 500) -> None:  # pragma: no co
     f = go.Figure(data=[g_edges, g_nodes], layout=layout)
 
     return f
+
+
+def plot_points(R, sample, size: int = 500):  # pragma: no cover
+    """Creates a plotly plot of the input points given their coordinates, highlighting an input
+    sample of points.
+
+    **Example usage**:
+
+    >>> R = np.random.normal(0, 1, (50, 50))
+    >>> sample = [1]*10 + [0]*40
+    >>> plot_points(R, sample).show()
+
+    .. image:: ../../_static/normal_pp.png
+       :width: 40%
+       :align: center
+       :target: javascript:void(0);
+
+    Args:
+        R (np.array): Coordinate matrix. Rows of this array are the coordinates of the points.
+        sample (list): subset of points to be highlighted, usually obtained from a sample.
+
+    Returns:
+         Figure: Plotly figure for subgraph
+    """
+    try:
+        import plotly.graph_objects as go
+    except ImportError:
+        raise ImportError("Plotly required for using plot(). Can be installed using pip install "
+                          "plotly or visiting https://plot.ly/python/getting-started/#installation")
+    except RuntimeError:
+        print("Plotly unable to open display")
+        raise
+
+    layout = go.Layout(
+        showlegend=False,
+        hovermode="closest",
+        xaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
+        yaxis=dict(showgrid=False, zeroline=False, showticklabels=False),
+        margin=dict(b=0, l=0, r=0, t=25),
+        height=size,
+        width=size,
+        plot_bgcolor="white",
+    )
+
+    points = go.Scatter(x=R[:, 0], y=R[:, 1],
+                        mode="markers",
+                        hoverinfo="text",
+                        marker=dict(color='#F2F2F2', size=30,
```

Comment:
Can we make the marker size to depend on the number of points? i.e. the points become smaller when the number of points increases such that they never overlap.

Reply:
So that the diameter of a point is always equal to the minimum distance between points?

Reply:
Ok, I will test this more

Reply:
@trbromley yes, something like that.